foundry.toml

```diff
@@ -0,0 +1,45 @@
+# Full reference https://github.com/foundry-rs/foundry/tree/master/config
+
+[profile.default]
+auto_detect_solc = false
+block_timestamp = 1_680_220_800 # March 31, 2023 at 00:00 GMT
+bytecode_hash = "none"
+cbor_metadata = false
+evm_version = "paris"           # See https://www.evmdiff.com/features?name=PUSH0&kind=opcode
+fuzz = { runs = 1_000 }
+gas_reports = ["*"]
+libs = ["lib"]
+optimizer = true
+optimizer_runs = 10_000
+out = "out"
+script = "script"
+solc = "0.8.21"
```

Comment:
By using this solc version, devs won't be able to publish to some rollups (that don't support the PUSH0 opcode yet).

Reply:
this was using `0.8.17` originally but you sugested a later version, but i think this was pre arbitrum. regardless will switch back